hadoop-hdds/docs/config.yaml

```
@@ -37,3 +37,21 @@ markup:
   goldmark:
     renderer:
       unsafe: true
+security:
+  enableInlineShortcodes: false
+  exec:
+    allow:
+      - ^dart-sass-embedded$
+      - ^go$
+      - ^npx$
+      - ^postcss$
+    osEnv:
+      - (?i)^(PATH|PATHEXT|APPDATA|TMP|TEMP|TERM)$
+  funcs:
+    getenv:
+      - ^HUGO_|^OZONE_VERSION$
```

I would replace regex "or" with yaml list:

```suggestion
  funcs:
    getenv:
      - ^HUGO_
      - ^OZONE_VERSION$
```

I think it will be friendlier for `git diff` in case changes are needed in the future.